### PR TITLE
Add SocketIO at hand cards and pushed cards

### PR DESCRIPTION
**Pull request by khamraeva**
- added  SocketIO, so all "players" can see the cards that other players pushed;
- added some cards to DB;
- changed logic of serving cards to players (it`ll be changed after demo);

### CODE REVIEW

File: src/web/frontend/components/Header/UserNav/UserNav.tsx [hunk 1]
```diff
@@ -13,7 +13,7 @@ function UserNav() {
     return (
         <UserProvider.context.Consumer>{context => (
             <React.Fragment>
-                {context.user && context.user.authenticated
```
Comment (takvol) [inline src/web/frontend/components/Header/UserNav/UserNav.tsx]: should keep `context.user && context.user.authenticated`

File: src/web/backend/authentication/authentication.controller.ts [hunk 2]
```diff
@@ -16,7 +16,7 @@ import { LoginUserData, JwtPayload } from './helpers';
 import EmailNotConfirmedException from '../exceptions/EmailNotConfirmedException';
 import { Player } from '../entities/Player';
 import { getRepository } from 'typeorm';
-import { uniqueNamesGenerator, Config as NamesConfig, adjectives, animals } from 'unique-names-generator';
+import { uniqueNamesGenerator, Config as NamesConfig, adjectives, animals, starWars } from 'unique-names-generator';
```
Comment (takvol) [inline src/web/backend/authentication/authentication.controller.ts]: starwars not needed, not critical though

File: src/web/frontend/components/Header/UserNav/UserNav.tsx (hunk 1 above)
Comment (khamraeva) [reply src/web/frontend/components/Header/UserNav/UserNav.tsx]: done

File: src/web/backend/authentication/authentication.controller.ts (hunk 2 above)
Comment (khamraeva) [reply src/web/backend/authentication/authentication.controller.ts]: done

**Pull request merged**